docs/reference/metrics.rst

```diff
@@ -0,0 +1,6 @@
+=======
+Metrics
+=======
+
```

Comment:
@annastuchlik @amnonh This page would benefit from an introduction to guide the reader. Consider adding information on what the user sees on this page, and instructions on how they can read and interact with these metrics.